docs/recipes/docker-server/README.md

```diff
@@ -64,5 +64,17 @@ kubectl get service
 # kubernetes    ClusterIP      10.X.X.X       <none>         443/TCP        9m
 # lhci-server   LoadBalancer   10.X.X.X       X.X.X.X        80:XXXXX/TCP   2m
 ```
+### Docker-compose
+
+An example docker-compose.yml could look like this:
```

Comment:
an explanation of why you'd want to use docker-compose for this would be great. 

AFAICT, there's no real benefit to using it because the lhci server is just a single docker container and there's not really much to orchestrate. `docker container run --publish 9001:9001 --mount='source=lhci-data,target=/data' --detach lhci` accomplishes the same task, no?

Reply:
Depends in yoyr setup, but indeed there's no benefit for it, the docker run and docker compose do the same thing. I use docker compose to keep all my running containers in the same file and not having to run a bunch of docker runs on startup.